iceoryx_hoofs/platform/linux/include/iceoryx_hoofs/platform/acl.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+#ifndef IOX_HOOFS_LINUX_PLATFORM_ACL_HPP
+#define IOX_HOOFS_LINUX_PLATFORM_ACL_HPP
```

Comment:
why is this whole file marked as new

Reply:
TBH, I have no clue :( I've moved all the files with the same `git mv` command. I know that Git has some magic detection mechanisms to determine how many percent of a file has changed. I don't understand why this worked with the other platform files, though.

Double checked, no duplicates around:
```
./iceoryx_hoofs/platform/qnx/include/iceoryx_hoofs/platform/acl.hpp
./iceoryx_hoofs/platform/mac/include/iceoryx_hoofs/platform/acl.hpp
./iceoryx_hoofs/platform/linux/include/iceoryx_hoofs/platform/acl.hpp
./iceoryx_hoofs/platform/win/include/iceoryx_hoofs/platform/acl.hpp
```

Reply:
Did you call `git mv iceoryx_utils iceoryx_hoofs` or did you do it for every single file?

Reply:
I think this is just the github heuristic which is doesn't recognize that it is an old file. I looked at the history of the file and it is preserved

Reply:
The former. Maybe I did something wrong when adding the files.